### PR TITLE
MTDSA-706 Added update of dividends

### PR DESCRIPTION
**Pull request by alexec**

### CODE REVIEW

File: public/api/conf/1.0/docs/dividends.md [hunk 1]
```diff
@@ -0,0 +1 @@
+TODO - QA API Developer Hub down at time of writing.
```
Comment (alexec) [inline public/api/conf/1.0/docs/dividends.md]: TODO

**Comment by saurabharora80** [comment]
`{
  "incomes": {
    "fromUKCompanies": 1000.25,
    "fromOtherUKSources": 1000.25
  }
}`
why do we need this unnecessary 'incomes' wrapper?

**Comment by saurabharora80** [comment]
what happens when the user does 
GET /self-assessment/ni/{nino}/dividends/{taxYear} 
before
PUT /self-assessment/ni/{nino}/dividends/{taxYear} ??

**Comment by saurabharora80** [comment]
Business question: are we sure Business wants to collect dividends totals for a year rather than Dividend totals per source for Dividend? 
i.e 

PUT /self-assessment/ni/{nino}/dividends/{taxYear}
OR 
PUT /self-assessment/ni/{nino}/dividends/{sourceId}/{taxYear}

**Pull request closed without merging**